warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/ShardedTableMapFile.java

```diff
@@ -54,7 +52,7 @@
     public static final String SHARDED_MAP_FILE_PATHS_RAW = "shardedMap.file.paths.raw";
     public static final String SHARD_VALIDATION_ENABLED = "shardedMap.validation.enabled";
     public static final String MAX_SHARDS_PER_TSERVER = "shardedMap.max.shards.per.tserver";
-    
+
```

Comment:
These whitespace changes suggest this code has not been compiled, since the formatter that runs during compilation is what adds the blank space at the beginning of the line. (I confirmed by checking out and building this PR--I'm left with an unclean tree.) Please make sure you've built the code and check in any auto-formatter changes.

Reply:
It should be fixed now with bed8dae7604e15c67ce5ec106928842629b3eb1d.  FYI I used "mvn -Pdev -Ddeploy -Dtar -DskipTests clean install"

Reply:
It's a good idea to build with `-Ddist` before submitting a PR too, so doc is generated and you'll discover if there are any doc errors.

Reply:
Build is also passing with -Ddist